src/main/java/com/passbook/sparkeighteen/peristence/POJO/SignUpRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.passbook.sparkeighteen.peristence.POJO;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotEmpty;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class SignUpRequest {
+
+    @NotEmpty(message = "email is mandatory")
```

Comment:
Can you explain the significance of @Empty

Reply:
@NotEmpty - Valid as long as its not null and its size is greater than zero.